src/components/Navigation.js

```diff
@@ -2,20 +2,18 @@ import React from 'react';
 import { Link } from 'react-router-dom';
 import './Navigation.css';
 
+const person = 'https://img.icons8.com/external-those-icons-flat-those-icons/24/000000/external-User-users-those-icons-flat-those-icons-4.png';
 const Navigation = () => (
-  <div className="container" data-testid="Navigation">
-    <h1>Bookstore CMS</h1>
-    <div className="navbar">
-      <div className="navitem">
-        <Link className="item" to="/">
-          BOOKS
-        </Link>
-        <Link className="item" to="/Category">
-          CATEGORIES
-        </Link>
-      </div>
-    </div>
-  </div>
+  <nav className="nav-bar" data-testid="Navigation">
+    <a className="nav-brand" href="#/">Bookstore CMS</a>
+    <ul className="nav-links">
+      <Link className="nav-link active-link" to="/" aria-current="page">BOOKS</Link>
+      <Link className="nav-link" to="/Category">CATEGORIES</Link>
+    </ul>
```

Comment:
- [ ] The same happens here, you're defining a list, but its direct children are `Link` components, in order for this to be valid HTML, you need to wrap those links in `<li>` elements. 🤓

Reply:
<li> tag add in nav links